lib/v2/rsshub/transform/sitemap.js

```diff
@@ -0,0 +1,52 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const config = require('@/config').value;
+
+module.exports = async (ctx) => {
+    if (!config.feature.allow_user_supply_unsafe_domain) {
+        ctx.throw(403, `This RSS is disabled unless 'ALLOW_USER_SUPPLY_UNSAFE_DOMAIN' is set to 'true'.`);
+    }
+    const { url } = ctx.params;
+    const response = await got({
+        method: 'get',
+        url,
+    });
+
+    const routeParams = new URLSearchParams(ctx.params.routeParams);
+    const $ = cheerio.load(response.data, { xmlMode: true });
+
+    const rssTitle = routeParams.get('title') ? routeParams.get('title') : ($('urlset url').length && $('urlset url').first().find('loc').text() ? $('urlset url').first().find('loc').text() : 'Sitemap');
+
+    let items;
+    const urls = $('urlset url').toArray();
+    if (urls && urls.length) {
+        items = urls.map((item) => {
+            try {
+                const title = $(item).find('loc').text() || '';
+                const link = $(item).find('loc').text() || '';
+                const description = $(item).find('image\\:image image\\:caption').text() || '';
```

Comment:
According to [the spec](https://www.sitemaps.org/protocol.html#xmlTagDefinitions), this tag does not exist.